openam-oauth2/src/main/java/org/forgerock/oauth2/core/ResourceOwnerSessionValidator.java

```diff
@@ -12,26 +12,31 @@
  * information: "Portions copyright [year] [name of copyright owner]".
  *
  * Copyright 2014-2016 ForgeRock AS.
+ * Portions Copyright 2023 Wren Security.
  */
 
 package org.forgerock.oauth2.core;
 
 import static com.sun.identity.shared.DateUtils.stringToDate;
 import static org.forgerock.oauth2.core.Utils.isEmpty;
 import static org.forgerock.oauth2.core.Utils.splitResponseType;
-import static org.forgerock.openam.oauth2.OAuth2Constants.Custom.*;
```

Comment:
Please update imports according to our style guide (single block for static imports and single block for non-static imports).

Reply:
I had an invalid import order configuration in in my IDE, this is now fixed.